tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_column_layout.py

```diff
@@ -30,7 +30,9 @@ def test_homescreen_column_layout(self):
         settings.launch()
 
         homescreen_settings = settings.open_homescreen()
-        homescreen_settings.select_icon_layout('Four Columns')
+        homescreen_settings.select_change_icon_layout()
```

Comment:
Ah, the reason I broke up the test like this is that if I bundle them together, I can't take screenshots in between.  What I'll do is create a method as you mentioned above, but make its components also available to outside as well.